src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/AppDesignerFolderProjectTreePropertiesProvider.cs

```diff
@@ -58,11 +58,11 @@ public void UpdateProjectTreeSettings(IImmutableDictionary<string, IProjectRuleS
             //
             // TODO: Read these default values from the rules themselves
             // See: https://github.com/dotnet/roslyn-project-system/issues/209
-            string folderName = ruleSnapshots.GetPropertyOrDefault(AppDesigner.SchemaName, AppDesigner.FolderNameProperty, "Properties");
-            string contextsVisibleOnlyInShowAllFiles = ruleSnapshots.GetPropertyOrDefault(AppDesigner.SchemaName, AppDesigner.ContentsVisibleOnlyInShowAllFilesProperty, "false");
+            string? folderName = ruleSnapshots.GetPropertyOrDefault(AppDesigner.SchemaName, AppDesigner.FolderNameProperty, "Properties");
```

Comment:
won't this never be null because it will be set to `"Properties"` if there is no value?

Reply:
Is there a way to flow through the nullness of the default value?